src/object_properties.h

```diff
@@ -54,6 +54,7 @@ struct ObjectProperties
 	std::string infotext;
 	//! For dropped items, this contains item information.
 	std::string wield_item;
+	bool no_static_save = false;
```

Comment:
What's the benefit of that?
This is a new feature, so older clients might not send this, in which case it would become false.